ccpp/data/GFS_typedefs.meta

```diff
@@ -4858,17 +4858,24 @@
   dimensions = ()
   type = integer
 [spp_prt_list]
-  standard_name =magnitude_of_spp_perturbations
+  standard_name = magnitude_of_spp_perturbations
   long_name = magnitude of spp perturbations
   units = 1
-  dimensions = (number_of_spp_schemes_perturbed)
+  dimensions = (number_of_perturbed_spp_schemes)
+  type = real
+  kind = kind_phys
```

Comment:
@JeffBeck-NOAA I think that the following should fix the GNU issue that was reported:
For each SPP variable that is conditionally allocated in GFS_typedefs.F90, (spp_prt_list, spp_stddev_cutoff, spp_var_list), please add the following metadata attribute
`active = (do_stochastically_perturbed_parameterizations)`

Reply:
Thanks @grantfirl!  I ended up having to apply this to spp_prt_list, spp_var_list, and spp_stddev_cutoff, but it fixed the problem!